package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@digicatapult/vitalam-service-template",
-  "version": "0.0.9",
+  "name": "@digicatapult/openapi-service-template",
+  "version": "0.0.1",
   "description": "Service for VITALam",
```

Comment:
@mattdean-digicatapult It looks like the workflow misses a reference? Is this because this fields needs defining manually?